salt/elife-dashboard/scheduler.sls

```diff
@@ -5,11 +5,17 @@ install-{{ app.name }}:
     git.latest:
         - name: ssh://git@github.com/elifesciences/{{ app.name }}
         - identity: {{ pillar.elife.projects_builder.key or '' }}
+
         # note: elife-article-scheduler is always deployed as master
         # until it has its own instance, we cannot a revision
         # using build vars
-        - rev: {{ salt['elife.cfg']('project.branch', 'master') }}
-        - branch: {{ salt['elife.cfg']('project.branch', 'master') }}
+        # update 2017-10-30: the below aren't working as we expected because the
+        # values for project.branch on --continuumtest, --prod and --end2end 
+        # are 'develop'!
+        #- rev: {{ salt['elife.cfg']('project.branch', 'master') }}
+        #- branch: {{ salt['elife.cfg']('project.branch', 'master') }}
+        - rev: master
+        - branch: master
```

Comment:
Indeed, these project values refer to the branch of `elife-dashboard`, while `project.rev` is its commit. Since we can't manage the version of two different projects on the same EC2 instance, let's pin `master` instead which is what this was supposed to do.
If we need to pin `elife-article-scheduler` to something, the standard practice would be to put a `elife-article-scheduler.sha1` file in `elife-dashboard` that says which commit to use. But if it comes to that I will merge it into the `elife-dashboard` project with my bare hands.